boards/microbit_v2/Makefile

```diff
@@ -25,8 +27,18 @@ flash-debug: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).elf
 flash: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
 	$(OPENOCD) $(OPENOCD_OPTIONS) -c "program $<; verify_image $<; reset; shutdown;"
 
-.PHONY: program
-program: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).bin
+.PHONY: flash-app
+flash-app: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).bin
 	arm-none-eabi-objcopy --update-section .apps=$(APP) $(KERNEL) $(KERNEL_WITH_APP)
 	$(OPENOCD) $(OPENOCD_OPTIONS) -c "program $(KERNEL_WITH_APP); verify_image $(KERNEL_WITH_APP); reset; shutdown;"
```

Comment:
do we still need this command if there is tockloader support? Wouldn't it make more sense to just install apps using tockloader?

Reply:
I agree, particularly since board autodetection has improved significantly in the next release of tockloader.

Reply:
I modified the Makefile. I am still having trouble using tockloader directly without the serial bootloader. With tockloader 1.6.0 I still have to specify the `--openocd`  parameter and the `-- board` parameter.